tap/nzyme-tap.conf.example

```diff
@@ -2,6 +2,8 @@
 leader_secret = "your_key_copied_from_nzyme_web_interface"
 leader_uri = "https://nzyme.example.org:22900/"
 accept_insecure_certs = false
+wifi_pkt_buffer_capacity = 65535
+ethernet_pkt_buffer_capacity = 65535
```

Comment:
Two thoughts:

* I think those should go into the `performance` section of the config file
* I would rename them to `wifi_broker_buffer_capacity` and `ethernet_broker_buffer_capacity`. This way, we have more flexibility for future buffer capacity configurations (like the processor buffers for `dns` or `arp`)

What do you think?

Reply:
done.